scripts/InitialMintAndLock.s.sol

```diff
@@ -0,0 +1,360 @@
+            numberOfVotingEscrow: 5,
+            amountPerVotingEscrow: TWO_MILLION,
+            lockTime: FOUR_YEARS
+        });
+
+        _batchInitialMintAndLock({
+            owner: FLOW_VOTER_EOA,
+            numberOfVotingEscrow: 3,
+            amountPerVotingEscrow: FOUR_MILLION,
+            lockTime: FOUR_YEARS
+        });
+
+        // 2. Mint to team members
+        _batchInitialMintAndLock({
+            owner: DUNKS,
+            numberOfVotingEscrow: 1,
+            amountPerVotingEscrow: FOUR_MILLION,
+            lockTime: FOUR_YEARS
+        });
+
+        _batchInitialMintAndLock({
+            owner: T0RB1K,
+            numberOfVotingEscrow: 3,
+            amountPerVotingEscrow: FOUR_MILLION,
+            lockTime: FOUR_YEARS
+        });
+
+        _batchInitialMintAndLock({
+            owner: CEAZOR,
+            numberOfVotingEscrow: 3,
+            amountPerVotingEscrow: FOUR_MILLION,
+            lockTime: FOUR_YEARS
+        });
+
+        _batchInitialMintAndLock({
+            owner: MOTTO,
+            numberOfVotingEscrow: 3,
+            amountPerVotingEscrow: FOUR_MILLION,
+            lockTime: FOUR_YEARS
+        });
+
+        _batchInitialMintAndLock({
+            owner: COOLIE,
+            numberOfVotingEscrow: 3,
+            amountPerVotingEscrow: FOUR_MILLION,
+            lockTime: FOUR_YEARS
+        });
+
+        // 3. Mint for future partners
+        _batchInitialMintAndLock({
+            owner: ASSET_EOA,
+            numberOfVotingEscrow: 3,
+            amountPerVotingEscrow: FOUR_MILLION,
+            lockTime: FOUR_YEARS
+        });
+
+        _batchInitialMintAndLock({
+            owner: TEAM_MULTI_SIG,
+            numberOfVotingEscrow: 13,
+            amountPerVotingEscrow: FOUR_MILLION,
+            lockTime: FOUR_YEARS
+        });
+
+        _batchInitialMintAndLock({
+            owner: TEAM_MULTI_SIG,
+            numberOfVotingEscrow: 3,
+            amountPerVotingEscrow: TWO_MILLION,
+            lockTime: FOUR_YEARS
+        });
+
+        _batchInitialMintAndLock({
+            owner: TEAM_MULTI_SIG,
+            numberOfVotingEscrow: 15,
+            amountPerVotingEscrow: TWO_MILLION,
+            lockTime: FOUR_YEARS
+        });
+
+        _batchInitialMintAndLock({
+            owner: TEAM_MULTI_SIG,
+            numberOfVotingEscrow: 16,
+            amountPerVotingEscrow: ONE_MILLION,
+            lockTime: FOUR_YEARS
+        });
+
+        _batchInitialMintAndLock({
+            owner: ASSET_EOA,
+            numberOfVotingEscrow: 5,
+            amountPerVotingEscrow: ONE_MILLION,
+            lockTime: TWO_YEARS
+        });
+
+        _batchInitialMintAndLock({
+            owner: ASSET_EOA,
+            numberOfVotingEscrow: 5,
+            amountPerVotingEscrow: ONE_MILLION,
+            lockTime: ONE_YEARS
+        });
+
+        _singleInitialMintAndLock(0xd0cC9738866cd82B237A14c92ac60577602d6c18, 1200000000000000000);
```

Comment:
what are these? and what do they represent as human readable numbers? What will they be used for?